detect_location_weather.py

```diff
@@ -15,16 +15,18 @@
 def location():
 
     # get the remote address of the client
-    user_ip = request.environ['REMOTE_ADDR']
-    print(user_ip)
+    if 'X-Forwarded-For' in request.headers:
```

Comment:
FYI, I think this entire file needs to be deleted and is a remnant but no longer in use by the website.